cfn-resources/cloud-backup-restore-jobs/docs/synchronouscreationoptions.md

```diff
@@ -48,7 +48,7 @@ _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormati
 
 #### ReturnSuccessIfTimeOut
 
-if set to true, the process will return success, in the event of a timeOut
+if set to true, the process will return success, in the event of a timeOut default false
```

Comment:
```suggestion
if set to true, the process will return success, in the event of a timeOut, default false
```

Reply:
small detail: lets adjust this in the json schema description as well to avoid having it modified each time we run `make build`